### ISSUE TITLE
data(topoWorld) should have -180<longitude<180

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#724]
@richardsc I have started work in ``create_data/topoWorld`` and I wonder if you could try running the two scripts there, and then try some of your code? I ask because I had to rewrite the code to create the dataset. There are some subtle features on lon and lat that I found in their documentation file, and I don't know if they were obeyed in the previous ``topoWorld`` dataset.

I don't know when you might be reading this, and am likely to work on this through the weekend.  So below is a checklist of my progress (since I basically hate the GH comment system and don't want to have a long list of comments that necessitate page scrolling just for one sentence in each comment).

* [x] reacquire raw binary data
* [x] write code to read these data (in ``create_data/topoWorld``)
* [x] invite CR to have a look at the new code (in ``create_data/topoWorld``)
* [x] alter code to use +/- longitude convention
* [x] incorporate into oce
* [x] detailed tests on a few cases (putting into ``tests/testhat`` as appropriate)
* [x] test in OAR
* [ ] test with daily usage over a few weeks

**Comment by dankelley** [comment]
I think this is working now in ``develop`` (commit e3d66cb36cf16033d306c1996ddd12ceadcb9227) but I will leave it open for a day.

**Comment by richardsc** [comment]
I had a look at the new `topoWorld` scripts, and everything looks good to me. FWIW I did some "on the fly" comparisons of the old and new `topoWorld` and it seems like the old one was shifted too far North. This is not a very reproducible (or robust) example, but I compared it with a hi-res Nordic Seas topography I made for a paper that used the full ETOPO5 data. The black line is my ETOPO5, the red line is the old `topoWorld`, and the green line is the new `topoWorld`. 
![screen shot 2015-08-22 at 21 44 27](https://cloud.githubusercontent.com/assets/233584/9426539/959d600c-4917-11e5-93c4-582484748f4a.png)

Interestingly, the code I used to generate my Nordic Seas data from the ETOPO5 file is not identical to yours, but I believe is functionally identical (see the lines with `seq()`):
```r
library(oce)

f <- file("../raw/ETOPO5.DAT", "rb")
nx <- 12 * 360
ny <- 12 * 180
z <- t(matrix(readBin(f, "integer", size=2, endian="big", n=nx*ny), 
                   byrow=TRUE, nrow=ny))
z <- z[,ny:1]
close(f)
dl <- 1/12
latitude <- seq(-90+dl, 90, length.out=ny)
longitude <- seq(0, 360-dl, length.out=nx)
```
Compared to your lines in `02_create_data.R`:
```r
latitude <- seq(90, by=-1/12, length.out=nlat)
longitude <- seq(0, by=1/12, length.out=nlon)
```
